types/testingutils/compare.go

```diff
@@ -147,7 +162,7 @@ func CompareSignedSSVMessageOutputMessages(t *testing.T, expectedMessages []*typ
 	for index, msg := range broadcastedSignedMsgs {
 		r1, _ := msg.GetRoot()
 
-		msg2 := broadcastedSignedMsgs[index]
+		msg2 := expectedMessages[index]
```

Comment:
why did this change?

Reply:
Because `broadcastedSignedMsgs` was being compared to `broadcastedSignedMsgs`, and it's wrong, no?